README.md

```diff
@@ -12,15 +12,23 @@ The diagram below shows the various components of ICAT and how they communicate
 
 ## Installation  
 
-Docker is needed to run the ICAT suite of micro-services.
+[Docker](https://www.docker.com/) is needed to run the ICAT suite of micro-services (see this information on [getting started with Docker](https://www.docker.com/get-started)).
 
-Google Chrome is needed to be able to log into Topcat.
+A browser (e.g. Google Chrome) is needed to be able to log into Topcat.
```

Comment:
Hi, for the time being, Google Chrome is explicitly needed. I haven't got topcat to work with Firefox yet. It's something to do with the way the 2 browsers address http (unsecure) connections.

Reply:
ok, thanks @moonraker595 - I'll change it back